lib/treat-locks/npm.lock.js

```diff
@@ -60,6 +59,9 @@ function treatLockFields(folderPath, outputFolder, fileName, npmOptions) {
   if (fs.existsSync(filePath, 'utf8')) {
     const json = JSON.parse(fs.readFileSync(filePath, 'utf8'));
     treatFields(json.dependencies, npmOptions);
+    if (json.packages !== undefined) {
```

Comment:
I prefer `if(![null, undefiend].includes(json.packages))` but again up to you

Reply:
I think we should stick with the undefined check, as it's either going to be defined (v2) or not (v1) in a lock file.